### PR TITLE
cts CharBuf selection is based on buffer with the max capacitance

### PR DESCRIPTION
**Pull request by habibayassin**
Signed-off-by: habibayassin <habibayassin@aucegypt.edu>

### CODE REVIEW

File: src/cts/src/TechChar.cpp [hunk 1]
```diff
@@ -484,9 +484,33 @@ void TechChar::initCharacterization()
...
+  for (int i = 0; i < masterVector.size(); i++){
+      float maxCap= 0.0;
+      bool maxCapExist= false;
+      odb::dbMaster* buf = db_->findMaster(masterVector[i].c_str());
+      sta::Cell* masterCell = db_network_->dbToSta(buf);
+      sta::LibertyCell* libertyCell = db_network_->libertyCell(masterCell);
+      sta::LibertyPort *input, *output;
+      libertyCell->bufferPorts(input, output);
+      output->capacitanceLimit(sta::MinMax::max(), maxCap, maxCapExist);
+      if( maxCap >= maxBuffCap){
```
Comment (maliberty) [inline src/cts/src/TechChar.cpp:498]: You need to have !maxCapExists by using the library default as is done in the code further down.

**Review by maliberty — CHANGES_REQUESTED** [review]

**Review by maliberty — APPROVED** [review]

**Review by maliberty — APPROVED** [review]

**Comment by maliberty** [comment]
I should have mentioned DCO to you.  When you commit you need to use '-s' to sign your commits.  I use a git alias for this so I don't forget.  Please click the Details link by DCO to get instructions on how to fix this up.

**Comment by habibayassin** [comment]
I followed the instruction in the Details link by DCO. Is there anything else I should do?

**Comment by maliberty** [comment]
Something seems to have gone quite wrong - you have a ton of conflicts in unrelated code.

**Comment by habibayassin** [comment]
yes I have noticed, trying to solve the problem

**Comment by QuantamHD** [comment]
@maliberty You need to kick off the scan workflow.

**Pull request merged**